src/main/java/uk/gov/hmcts/sptribs/caseworker/util/EventUtil.java

```diff
@@ -89,4 +90,18 @@ public static String getMainContent(DecisionTemplate decision) {
         return mainContent;
     }
 
+    public static String getOrderMainContent(OrderTemplate order) {
+        String mainContent = "";
+        if (order.equals(OrderTemplate.CIC7_ME_DMI_REPORTS)) {
+            mainContent = DocmosisTemplateConstants.ME_DMI_MAIN_CONTENT;
```

Comment:
You could use a switch statement here?
OrderTemplate is an enum so you can use == instead of .equals, which has the advantage of compile-time type checking.